diff --git a/hub/src/app.py b/hub/src/app.py
index 3a121c0..0de0568 100644
--- a/hub/src/app.py
+++ b/hub/src/app.py
@@ -127,8 +127,7 @@ def rule5(env: Environment):
     print("Rule 5")
 
     target_temp = 40
-    # // BUG TO UNPATCH forgot to add abs
-    dif = abs(env.data["windwow"]["temperature"] - 10)
+    dif = env.data["windwow"]["temperature"] - 10
     if env.data["windwow"]["temperature"] < 10:
         target_temp = dif * 3 + target_temp
 
