diff --git a/apps/smartkettle/smart_kettle.cpp b/apps/smartkettle/smart_kettle.cpp
index a126d23..3684d18 100644
--- a/apps/smartkettle/smart_kettle.cpp
+++ b/apps/smartkettle/smart_kettle.cpp
@@ -128,11 +128,6 @@ namespace Generic {
             string settingsMessage;
             // if temperature is set to 0 degrees then use the last value - stateful
             if (temp == "0") {
-                if (boilHistory.size() == 0) {
-                    response.send(Http::Code::Bad_Request, Utils::getBadRequestResponse());
-                    return;
-                }
-
                 auto lastBoil = boilHistory[boilHistory.size() - 1];
                 temp = lastBoil.getTemperature();
                 scale = lastBoil.getScale();
