diff --git a/hub/app.py b/hub/app.py
index f1b3c71..193d180 100644
--- a/hub/app.py
+++ b/hub/app.py
@@ -181,8 +181,6 @@ def rule4(env: Environment):
             10 - env.data["windwow"]["luminosity"]/10 + brightness_base,
             brightness_base 
         )
-    # // BUG TO UNPATCH forget to call min
-    env.data["smarttv"]["brightness"] = min(env.data["smarttv"]["brightness"], 10)
     env.clients["smarttv"].set_brightness_level_post(int(env.data["smarttv"]["brightness"]))
 
 # If it's cold outside, increase the temperature in the kettle by 3 degrees for every degree under 10 at window's thermometer
