diff --git a/hub/app.py b/hub/app.py
index f1b3c71..0a411aa 100644
--- a/hub/app.py
+++ b/hub/app.py
@@ -165,8 +165,7 @@ def rule3(env: Environment):
     luminosity_sensor_id = 3
     threshold = env.settings["plant_lamp_window_treshold"]
 
-    # // BUG TO UNPATCH don't check window temperature
-    if env.data["windwow"]["luminosity"] < threshold and env.data["windwow"]["temperature"] < 30:
+    if env.data["windwow"]["luminosity"] < threshold:
         print(f"If luminosity amount is less than the threshold = {threshold}, then turn on the lamp")
         print("Lamp turned on")
         env.clients["flowerpower"].activate_solar_lamp_get()
